src/interface/widgets/PlaysWidget.cpp

```diff
@@ -11,9 +11,12 @@ namespace rtt::ai::interface {
     }
 
     void PlaysWidget::updatePlays() {
-        std::lock_guard mtxLck{ dataMtx };
```

Comment:
Why did you remove the mutex?

Reply:
I no longer modify an internal string / buffer, I just write to the textbox directly